django_ses/urls.py

```diff
@@ -1,8 +1,8 @@
 from django.urls import path
 
-from django_ses.views import dashboard
```

Comment:
If anyone is importing this view directly, it will break their code.

Do you think we should backfill this import, maybe add a `Deprecation` warning when importing `dashboard`, but it still works?

Otherwise I think we may need to bump a major version.

Reply:
Yes, I manually tested and confirmed that all was fine. We can add a Deprecation Warning.

Reply:
@EricOuma would you like to add? I don't see any updates since my review.

Reply:
Made the updates.

Reply:
I have tested both views and all is working fine